anndata/__init__.py

```diff
@@ -34,4 +34,24 @@
 )
 
 # backwards compat / shortcut for default format
-from ._io import read_h5ad as read
+from ._io import read_h5ad as read  # noqa: F401
+
+__all__ = [
+    "__version__",
+    "AnnData",
+    "concat",
+    "Raw",
```

Comment:
```suggestion
    "Raw",
    "experimental",
```

Right?

Reply:
No, if you have a submodule that nothing has imported yet, the submodule is not a member of the super module:

```console
$ mkdir mod
$ touch mod/submod.py
$ touch mod/__init__.py
$ python -c 'import mod; mod.submod'
AttributeError: Module `mod` has no member `submod`
$ python -c 'import mod.submod'  # works
```